src/plone/restapi/configure.zcml

```diff
@@ -5,10 +5,13 @@
     xmlns:plone="http://namespaces.plone.org/plone"
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
     xmlns:zcml="http://namespaces.zope.org/zcml"
+    xmlns:meta="http://namespaces.zope.org/meta"
     i18n_domain="plone.restapi">
 
   <i18n:registerTranslations directory="locales" />
 
+  <meta:provides feature="plonerestapi-7" />
```

Comment:
@sneridagh I am +1 on this. Though, we have to be super careful that we change this when we release p.restapi 8.

Reply:
To be sure: we should not _change_ this when we release plone.restapi 8. We should keep it and **add** a second feature`plonerestapi-8`. `CMFPlone` does the same, and still has features `plone-4`, `plone-52`, etc.
But maybe you mean that.